StaticAnalysisQualityGate/task.json

```diff
@@ -72,7 +72,7 @@
       }
     }
   ],
-  "instanceNameFormat": "Parasoft Static Analysis Quality Gate",
+  "instanceNameFormat": "Parasoft Static Analysis Quality Gate - $(type) - $(severity) - $(threshold)",
```

Comment:
It is shown like
![image](https://github.com/parasoft/parasoft-findings-vsts/assets/109124579/715f1370-6583-4f17-a8a1-313348ae3356)